src/Converter/Feature2Mesh.js

```diff
@@ -496,25 +557,33 @@ function featureToMesh(feature, options) {
     var mesh;
     switch (feature.type) {
         case FEATURE_TYPES.POINT:
-            mesh = featureToPoint(feature, options);
+            if (feature.style.point && feature.style.point.model && feature.style.point.model.object) {
+                mesh = pointsToInstancedMeshes(feature);
+            } else {
+                mesh = featureToPoint(feature, options);
+                mesh.material.vertexColors = true;
+                mesh.material.color = new THREE.Color(0xffffff);
```

Comment:
Maybe instead of repeating these for every case of `feature.type`, we could add 
```js
if (!feature.style.point?.model?.object) {
    mesh.material.vertexColors = true;
    mesh.material.color = new THREE.Color(0xffffff);
}
```
on line 586 just after the switch cases.

Another option would be to add 
```js
mesh.isInstancedMesh = true  // or some name like that
```
just after line 561 and then add 
```js
if (!mesh.isInstancedMesh) {
    mesh.material.vertexColors = true;
    mesh.material.color = new THREE.Color(0xffffff);
}
```
in line 586.

I know these solution are not perfect, but I wonder what's the best between adding one of these checks on line 586 or setting the same options with the same hard coded values multiple times. WDYT ?